syntax = "proto3";

message VehicleClassificationOptions {
    string kernel_name = 1;
    bool use_detections = 2;
    bool print_results = 3;
	int32 font = 4;
	double font_scale = 5;
	int32 font_thickness = 6;
	int32 box_thickness = 7;
	int32 line_type = 8;
}